### PR TITLE
introspect error object

### PR DESCRIPTION
**Pull request by ceonizm**
Hello, 

When launching bozon start, if something go wrong (typescript compilation for example) we obtain the following message on the output

> [bozon] Failed to build application ✖
> [bozon] [object Object]

Don't you think that using something like Util.inspect to display the content of the object would be more helpful to troubleshoot ?

### CODE REVIEW

**Comment by alchaplinsky** [comment]
yep, makes sense. Thanks!

File: src/builder/index.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { bundle } from 'builder/bundle'
 import { watch } from 'builder/watcher'
 import { source, sourcePath, destinationPath } from 'utils'
 import { log, warn, startSpinner, stopSpinner } from 'utils/logger'
+import * as Util from "node/util";
```
Comment (alchaplinsky) [inline src/builder/index.js:10]: import should be done as follows

```suggestion
import { inspect }  from 'util'
```

File: src/builder/index.js [hunk 2]
```diff
@@ -34,7 +35,7 @@ const onBuildSuccess = (config, env, warnings) => {
 
 const onBuildError = error => {
   stopSpinner(BUILD_FAILED, false)
-  log(chalk.grey(error))
+  log(chalk.grey(Util.inspect(error)))
```
Comment (alchaplinsky) [inline src/builder/index.js:38]:
```suggestion
  log(chalk.grey(inspect(error)))
```

File: src/builder/index.js (hunk 1 above)
Comment (ceonizm) [reply src/builder/index.js:10]: ok no problem for me :) 
I've committed your suggested changes

**Review by alchaplinsky — APPROVED** [review]

**Pull request merged**